app/src/main/java/eu/qm/fiszki/MainActivity.java

```diff
@@ -12,7 +12,7 @@
 
 
 public class MainActivity extends AppCompatActivity {
-    TimerClass timer = new TimerClass();
+    TimerClass Timer = new TimerClass();
```

Comment:
to w końcu jak to nazywamy? z dużej czy z małej :u5272: :u5408: :u55b6: :u6307: :u6708: